user_file_manager.py

```diff
@@ -0,0 +1,20 @@
+import os
+from datetime import datetime
+
+
+class UserFileManager():
+    def __init__(self):
+        self.root_dir = os.path.dirname(__file__)
+
+        self.history_dir = os.path.join(self.root_dir, "History")
+        if not os.path.exists(self.history_dir):
+            os.mkdir(self.history_dir)
```

Comment:
Please, add possibility to configure FileManager by passing output directory path to it.